src/validator.js

```diff
@@ -368,6 +383,11 @@
     Extended.prototype = Object.create( Assert.prototype );
     Extended.prototype.constructor = Extended;
 
+    // Copy all the static methods.
+    Object.keys( Assert ).forEach( function( key ) {
```

Comment:
This would break IE8 support. I'm not against or in favor specifically but if we made this choice we should update the README accordingly to specify supported browsers.